app/move/fields/create.js

```diff
@@ -112,7 +112,7 @@ function explicitAssessmentQuestion(name, value, conditional) {
 }
 
 const dateField = {
-  validate: ['date'],
+  validate: ['date', 'required'],
```

Comment:
This was removed from here as not every date field will necessarily be "required". I think the `required` validation should be done at an individual field level rather than this shared level for future safety.

Reply:
Fair enough, but for now at least they all are required. I am happy to revert it if you want, but we should code for now, not for an hypothetical future 😄

Reply:
Happy to leave if for now then. 

My thought was more that a shared date configuration should contain date based configuration and field specific configuration (ie whether that field is required or not on the step it is used) should be done at field level, not shared level.

Just felt weird to say every single date field now and forever will always be required. However, felt fine to say every date field should be validated as a date. Is more about logical separating validation and putting them in the more sensible point in the configuration.